playwright/_impl/_network.py

```diff
@@ -285,10 +285,13 @@ async def _race_with_page_close(self, future: Coroutine) -> None:
             # When page closes or crashes, we catch any potential rejects from this Route.
             # Note that page could be missing when routing popup's initial request that
             # does not have a Page initialized just yet.
+            fut = asyncio.create_task(future)
             await asyncio.wait(
-                [asyncio.create_task(future), page._closed_or_crashed_future],
+                [fut, page._closed_or_crashed_future],
                 return_when=asyncio.FIRST_COMPLETED,
             )
+            if page._closed_or_crashed_future.done():
+                await asyncio.gather(fut, return_exceptions=True)
```

Comment:
I was thinking a bit about that, isn't there a better way of doing this? Probably:
```
try:
    await fut
catch Exception:
    pass
```
Just out of curiosity.

Reply:
avoiding for pycharm `Too broad exception clause` lint warning, also it's a one-liner code. Personally, I prefer it.